PyQt5-stubs/QtCore.pyi

```diff
@@ -5994,6 +5994,10 @@ class QPoint(sip.simplewrapper):
 
     def __neg__(self) -> 'QPoint': ...
     def __pos__(self) -> 'QPoint': ...
+    def __add__(self, point: 'QPoint') -> 'QPoint': ...
+    def __sub__(self, point: 'QPoint') -> 'QPoint': ...
+    def __mul__(self, point: 'QPoint') -> 'QPoint': ...
```

Comment:
Multiplying a point by a point doesn't make any sense:

```
>>> QPoint(1, 2) * QPoint(1, 2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for *: 'QPoint' and 'QPoint'
```

This should probably be `factor: float` or so?

Reply:
Sorry, it a typo. Float is right.